pyfar/classes/audio.py

```diff
@@ -1011,6 +1025,41 @@ def power(data: tuple, domain='freq'):
     return _arithmetic(data, domain, _power)
 
 
+def matrix_multiplication(data: tuple, domain='freq'):
+    """Matrix multiplication along first two axes of pyfar audio objects
+    and array likes.
+
+    Pyfar audio objects are: :py:func:`Signal`, :py:func:`TimeData`, and
```

Comment:
Maybe add a hint on which function is used in the background (np.dot or np.matmul come to my mind).